dgraphtest/cluster.go

```diff
@@ -253,14 +253,14 @@ func (hc *HTTPClient) RunGraphqlQuery(params GraphQLParams, admin bool) ([]byte,
 
 // Backup creates a backup of dgraph at a given path
 func (hc *HTTPClient) Backup(c Cluster, forceFull bool, backupPath string) error {
-	// backup API was made async in the commit after this commit d3bf7b7b2786bcb99f02e1641f3b656d0a98f7f4
-	syncAPI, err := IsParent("caed2cb8c68b0ea8c69a3d71410a5fa6b0b94985", c.GetVersion())
+	// backup API was made async in the commit d3bf7b7b2786bcb99f02e1641f3b656d0a98f7f4
+	asyncAPI, err := IsParent("d3bf7b7b2786bcb99f02e1641f3b656d0a98f7f4", c.GetVersion())
```

Comment:
For different branches, the commit id might be different? Can we come up with a better way to check this?

Reply:
Yeah, we probably can but I am thinking let's improve this function when it stops working for us. Today it seems to work fine. When we see problems with this function, it will be easy to solve it too then.